ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/Main.java

```diff
@@ -109,7 +111,13 @@ public void start() throws ExecutionException, InterruptedException {
     var tokenIssuer = new TokenIssuerImpl(config.baseUri(), keyStore, codeRepo);
     var sessionRepo = buildSessionRepo(config.sessionStore(), meterRegistry);
 
-    var httpClient = HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(5)).build();
+    var sslContext = TlsContext.fromClientCertificate(config.federation().entitySigningKey());
```

Comment:
This adds the client certificate to the list of certificates to use for authentication.